mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
-// There are syntax errors in this code - can you fix it to pass the tests?
+// There are syntax errors in this code - can you fix it to pass the tests?//
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
-
+function introduceMe(name, age){
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
 function getTotal(a, b) {
-  total = a ++ b;
+  total = a + b;
```

Comment:
This code will work, but we should always use either let or const when declaring a new variable :sweat_smile: